docker/llm/finetune/lora/cpu/docker/Dockerfile

```diff
@@ -19,7 +19,7 @@ COPY --from=key-getter /root/intel-oneapi-archive-keyring.gpg /usr/share/keyring
 RUN echo "deb [signed-by=/usr/share/keyrings/intel-oneapi-archive-keyring.gpg] https://apt.repos.intel.com/oneapi all main " > /etc/apt/sources.list.d/oneAPI.list
 
 RUN mkdir /bigdl/data && mkdir /bigdl/model && \
-# install pytorch 2.0.1
+# install pytorch 2.1.0
```

Comment:
why 2.1.0? the below ipex version is still 2.0.1, which is inconsistent to torch 2.1.0

Reply:
I only edited comments, the torch version has been modified in this [commit](https://github.com/intel-analytics/BigDL/commit/30d14c51a62ea92db886de188ff63dd9ef6b8c91)